src/agent/structures/structures.ts

```diff
@@ -40,6 +41,9 @@ export interface Statistics {
   chats_count: number;
   threads_count: number;
   visits_count: number;
+  page_views_count: number;
+  greetings_shown_count: number;
+  greetings_Accepted_count: number;
```

Comment:
the same typo in all versions
```suggestion
  greetings_accepted_count: number;
```

Reply:
I'll use the comment to mark fixes in various versions:
Fixed for v3.5